src/app/alarm-menu/alarm-menu.component.scss

```diff
@@ -1,6 +1,3 @@
-@import '~@angular/material/theming';
```

Comment:
Does this need `@use '~@angular/material' as mat;` as per further down or is material just not used in alarm-menu?